pkg/cluster/cluster.go

```diff
@@ -397,6 +399,23 @@ func (os *ClusterSpec) Validate() error {
 	if s.InitMode == nil {
 		return fmt.Errorf("initMode undefined")
 	}
+	if len(s.AdditionalReplicationSlotNames) > 0 {
+		rp := regexp.MustCompile("[_0-9a-z]+")
+		symbolsNameFilter := regexp.MustCompile("[_0-9]")
+
+		for _, replSlotName := range s.AdditionalReplicationSlotNames {
+			if trimmedSlotName := strings.TrimSpace(replSlotName); len(trimmedSlotName) == 0 ||
```

Comment:
I think you can simplify it:

* Check using using `IsValidReplSlotName`: https://github.com/sorintlab/stolon/blob/master/pkg/postgresql/utils.go#L328 ?
* Check that the replslotname isn't a reserved stolon name using `IsStolonName`: https://github.com/sorintlab/stolon/blob/master/common/common.go#L65